website/src/pages/product-ecosystem.tsx

```diff
@@ -1,12 +1,93 @@
-import React from "react"
+import React, { useState } from "react"
+import ReactDOMServer from "react-dom/server"
 import Layout from "@theme/Layout"
-import styles from "./product-ecosystem.module.css"
+import Link from "@docusaurus/Link"
 import common from "../css/common.module.css"
-import { CheckCircle2 } from "lucide-react"
+import {
+  CheckCircle,
+  CheckCircle2,
+  FlagTriangleRight,
+  Layers,
+  LocateFixed,
+  Timer,
+} from "lucide-react"
 import CTA from "../components/cta/cta"
 import Image from "@theme/IdealImage"
 
+import ReactTooltip from "react-tooltip"
+
+import "react-responsive-modal/styles.css"
+import { Modal } from "react-responsive-modal"
+
+import * as RoadmapItems from "../util/RoadmapItems"
+
+// declaring last to easily override the styles of everything imported above
+import styles from "./product-ecosystem.module.css"
+
+// Tooltip Template -- moved out here bc <RoadmapColumn> (declared at bottom) uses it
+const formatTooltipContent = data => {
+  // ReactTooltip doesn't accept JSX, so have to format the content this way (as string)
+
+  if (!data) return ""
+
+  const name = data.title
+  const desc = data.text
+  const source = data.source
```

Comment:
You could use destructuring here.
```suggestion
  const {
    name,
    text: desc,
    source,
  } = data;
```

But what you have is fine.